File: Difficulty: Easy/Number of occurrence/number-of-occurrence.cpp
class Solution {
  public:
    int countFreq(vector<int>& arr, int target) {
        // code here
        int count = 0;
        int hash[1000001] = {0};
        for (int i=0; i<arr.size(); i++){
            hash[arr[i]]++;
        }
        
        return hash[target];
    }
};

File: Difficulty: Easy/Number of occurrence/number-of-occurrence.java
class Solution {
	int countFreq(int[] arr, int target) {
		// code here
		HashMap<Integer, Integer> map = new HashMap<>();
		for (int i = 0; i<arr.length; i++) {
			map.put(arr[i], map.getOrDefault(arr[i], 0) + 1);
		}
		Integer ans = map.get(target);
		return (ans != null) ? ans : 0;
	}
}
